src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultLSPEditorFeatureDetector.cs

```diff
@@ -90,7 +98,7 @@ public override bool IsLSPEditorFeatureEnabled()
                 return true;
             }
 
-            if (IsFeatureFlagEnabled())
+            if (IsFeatureFlagEnabledCached())
```

Comment:
This is a separate cached value for the editor feature flag. Just renaming to make it clear it's not taking a _live_ look of the value.